controllers/istio_controller_test.go

```diff
@@ -302,8 +304,9 @@ func newCondition(conditionType v1.IstioConditionType, status bool, reason v1.Is
 }
 
 func TestApplyProfile(t *testing.T) {
+	const version = "my-version"
```

Comment:
Are these changes from the other PR? (The one that applies the default profiles?)

Reply:
Nope. These are more related to this PR, since we're getting rid of the v3.0 version in this PR. But instead of using "latest" here, I intentionally used a mock version name, since it doesn't actually need to be any of the actual versions.